Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM openjdk:11-jre-slim
```

Comment:
`openjdk:11-jre-slim` is deprecated.
https://github.com/docker-library/openjdk/issues/505